themes/uncontained.io/package.json

```diff
@@ -1,37 +1,26 @@
 {
-  "name": "minimo",
-  "description": "Minimalist theme for Hugo",
-  "version": "2.0.0",
-  "author": {
-    "name": "Munif Tanjim",
-    "url": "https://muniftanjim.com"
-  },
-  "repository": "MunifTanjim/minimo",
-  "license": "MIT",
+  "name": "uncontained-theme",
+  "version": "0.1.0",
+  "description": "Theme for uncontained.io",
+  "main": "index.js",
   "scripts": {
-    "build": "NODE_ENV=production webpack -p",
-    "start": "hugo server --source=exampleSite --themesDir=../.. --disableFastRender",
-    "watch": "webpack --watch --progress --colors"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "gulp watch",
+    "build": "gulp",
+    "start": "gulp watch"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/redhat-cop/uncontained.io/tree/master/themes/uncontained.io"
```

Comment:
What happens with this URL? Do we need to consider support for multi-branch when we go to do pipelines?

Reply:
This URL is just there for humans who are reading the file and want to learn more. It's not used by any automated process, so we could change it down the road without breaking anything. ([More info on the git attribute in package.json](redhat-cop.github.io))